rpm/gratia-probe.spec

```diff
@@ -128,6 +128,8 @@ git_commit_id=$(gzip -d < %{SOURCE0} | git get-tar-commit-id)
     PROBE_DIR=$RPM_BUILD_ROOT/%{_sysconfdir}/gratia/$probe
     install -d $PROBE_DIR
     install -m 644 common/ProbeConfigTemplate.osg $PROBE_DIR/ProbeConfig
+    ln -s %{_sysconfdir}/gratia/$probe/ProbeConfig \
+          $RPM_BUILD_ROOT/%{_datadir}/gratia/$probe/ProbeConfig
```

Comment:
Could you add a short comment here explaining why?

Reply:
Done